RELEASING.md

```diff
@@ -42,3 +42,9 @@ To address this, all ChangeLogs for Nubis will be generated by [github_changelog
 The process is really simple and will look like this:
 
     $> github_changelog_generator --future-release v1.0.0 nubisproject/nubis-docs
+
+## Announcing
+
+Sometimes, like when making a new release, it can be desired to reach all Nubis Users. The current day-to-day mechanism to communicate with Nubis users is on the #nubis-users channel on irc.mozilla.org
+
+However, for more official announcements, or information that needs more reasonable delivery guarantees than IRC, we use the [https://groups.google.com/d/forum/nubis-announce](nubis-announce) distribution list. It can be reached at [nubis-announce@googlegroups.com](nubis-announce@googlegroups.com)
```

Comment:
Your link syntax is reversed here. It should be [nubis-announce](https://groups.google.com/d/forum/nubis-announce)